pkgs/by-name/cn/cnsprcy/package.nix

```
@@ -1,33 +1,51 @@
 {
-  lib,
+  lib, # Dependencies/arguments
```

I don't think we need comments, here, unless we're explaining some non-basic things like why a test is skipped or why we did some ambiguous things.

```suggestion
  lib,
```